app/helpers/service_helper.rb

```diff
@@ -90,10 +90,25 @@ def switch_banner_bottom_copy
     t("service_banner.bottom.#{service_choice}")
   end
 
-  def region_name
+  def service_region
     t("title.region_name.#{service_choice}")
   end
 
+  def replace_service_tags(content)
+    content.gsub %r{\[\[SERVICE_[A-Z_]+\]\]} do |match|
+      case match
+      when '[[SERVICE_NAME]]'
```

Comment:
I am thinking if we add the `[[` and `]]` in the regexp,
```
when 'SERVICE_NAME'
...
when 'SERVICE_PATH'
...
```
which is more readable

Reply:
After discussion decided this actually ends up more complicated (requires use of RegExp.last_match) so leaving as is